internal/apiutil/errors.go

```diff
@@ -5,7 +5,9 @@ package apiutil
 
 import "github.com/mainflux/mainflux/pkg/errors"
 
-// Errors used to distingate and log on API request validation
+// Errors defined in this file are used by the LoggingErrorEncoder decorator
+// to distingate and log API request validation errors and avoid that service
```

Comment:
To distinguish?

Reply:
@manuio Please fix this typo and we're ready to merge.

Reply:
ok!